app/controllers/monthly_reports_controller.rb

```diff
@@ -2,7 +2,9 @@ class MonthlyReportsController < ApplicationController
   def index
     references = [{ user: :groups }, :monthly_working_process, { monthly_report_tags: :tag }]
     @q = MonthlyReport.includes(references).ransack(search_params)
-    @monthly_reports = @q.result(distinct: true).released.order('shipped_at desc').page params[:page]
+    @monthly_reports = @q.result(distinct: true).released.order('shipped_at desc')
+    @monthly_reports = @monthly_reports.where(target_month: search_params[:target_month].to_date) if search_params.try(:[],:target_month).present?
```

Comment:
__[rubocop]__
* Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [147/128] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->